source/code/troubleshooter/modules/helpers.py

```diff
@@ -13,6 +12,16 @@
 except ImportError:
     from urllib2 import urlopen
 
+# linux_distribution() function was moved to distro for Python 3.8+
+try:
+    from platform import linux_distribution
+    from platform import dist
+except ImportError:
+    try:
+        from distro import linux_distribution
```

Comment:
is `distro` part of python stdlib?

Reply:
I'm not sure, but on an Ubuntu20 VM with Python 3.8 `import distro` works without any issues